julia-editor/script.js

```
@@ -0,0 +1,95 @@
+        if (line.trim().startsWith('#')) {
+          result += line + '\n';
+        } else if (line.trim().includes('println')) {
+          const match = line.match(/println\("(.+)"\)/);
+          if (match) result += match[1] + '\n';
+        } else if (line.trim().includes('+') || line.trim().includes('*')) {
+          try {
+            // Simple evaluation
+            const parts = line.split('=');
+            if (parts.length > 1) {
+              const expr = parts[1].trim();
+              const value = eval(expr);
```

Biome (2.5.3)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)